tests/integration/conftest.py

```diff
@@ -107,8 +106,8 @@ async def synapse_app_fixture(
     use_existing = pytestconfig.getoption("--use-existing", default=False)
     if use_existing:
         return model.applications[synapse_app_name]
-    if synapse_app_refresh_name in model.applications:
-        await model.remove_application(synapse_app_refresh_name, block_until_done=True)
+    if synapse_app_charmhub_name in model.applications:
+        await model.remove_application(synapse_app_charmhub_name, block_until_done=True)
```

Comment:
What would be the reason that the charm is already deployed in the model? I would have expected any previous tests to remove the application from the model before finishing?

Reply:
Thanks for the advice. I see no reason to remove the application in there. If at some point there are conflicts, the tests can be put in another module or change the scope for the fixture.

Removed that code.